docs/source/type_info.md

```diff
@@ -1,23 +1,24 @@
+```{eval-rst}
 .. currentmodule:: torch
+```
 
-.. _type-info-doc:
+(type-info-doc)=
+# Type Info
 
-Type Info
-=========
+The numerical properties of a {class}`torch.dtype` can be accessed through either the {class}`torch.finfo` or the {class}`torch.iinfo`.
 
-The numerical properties of a :class:`torch.dtype` can be accessed through either the :class:`torch.finfo` or the :class:`torch.iinfo`.
+(finfo-doc)=
 
-.. _finfo-doc:
-
-torch.finfo
------------
+## torch.finfo
 
+```{eval-rst}
 .. class:: torch.finfo
+```
 
-A :class:`torch.finfo` is an object that represents the numerical properties of a floating point
-:class:`torch.dtype`, (i.e. ``torch.float32``, ``torch.float64``, ``torch.float16``, and ``torch.bfloat16``). This is similar to `numpy.finfo <https://numpy.org/doc/stable/reference/generated/numpy.finfo.html>`_.
+A {class}`torch.finfo` is an object that represents the numerical properties of a floating point
+{class}`torch.dtype`, (i.e. ``torch.float32``, ``torch.float64``, ``torch.float16``, and ``torch.bfloat16``). This is similar to `numpy.finfo <https://numpy.org/doc/stable/reference/generated/numpy.finfo.html>`_.
```

Comment:
I think in markdown we only need 1 backtick ` rather than 2 backticks but 2 seems to still work in any case. Do we want to update these backticks as well?

Reply:
Happy to do so!

Reply:
Actually, in this example PR that was provided for the docathon, #154438, they kept 2 backticks. I will leave as is.